### PR TITLE
Adding API Management templates

### PR DESCRIPTION
**Pull request by miaojiang**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@miaojiang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mijiang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-azure-api-management-create/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Azure API Management
+
```
Comment (singhkay) [inline 101-azure-api-management-create/README.md]: Please see the sample readme https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md

File: 101-azure-api-management-create/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Azure API Management
```
Comment (singhkay) [inline 101-azure-api-management-create/README.md]: Need a better name than just a generic name

File: 101-azure-api-management-create/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters":
+    {
+        "publisherEmail":
```
Comment (singhkay) [inline 101-azure-api-management-create/azuredeploy.json]: Every parameter needs a metadata.description tag. Please review the contribution guide https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

**Comment by miaojiang** [comment]
Updated API management templates per feedback received

File: 101-azure-api-management-create/metadata.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "itemDisplayName": "Create an API Management instance using a template",
+  "description": "This template creates a developer instance of Azure API Management",
+  "summary": "Create an API Management instance using a template",
+  "githubUsername": "miaojiang",
+  "dateUpdated": "2016-04-22",
+  "publisherEmail":"apimgmt@microsoft.com"
```
Comment (singhkay) [inline 101-azure-api-management-create/metadata.json]: metadata file doesn't have this element in the schema

File: 101-azure-api-management-create/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+			"metadata": {
+				"description": "The email address of the owner of the service"
+			}
+        }
+        ,
+        "publisherName":
+        {
+            "type": "string",
+            "minLength": 1,
+			"metadata": {
```
Comment (singhkay) [inline 101-azure-api-management-create/azuredeploy.json]: Please format the JSON file

**Comment by singhkay** [comment]
@miaojiang left comments, thanks

**Comment by miaojiang** [comment]
Fixing templates per feedback

**Comment by singhkay** [comment]
@miaojiang Looks good and passed CI. One minor nit, can you name the folder 101-api-management-create, since the repo is already "azure". You'll also need to update any links which had the folder name in it

**Comment by miaojiang** [comment]
@singhkay Actually 'Azure API Management' is the commonly used name similar to 'Azure Search'.

**Comment by singhkay** [comment]
@miaojiang got it, merged

**Pull request merged**